tests/backup/backup_share_test.go

```diff
@@ -4215,3 +4215,310 @@ var _ = Describe("{SwapShareBackup}", func() {
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation and validation of backup [%s] of namespace (scheduled Context) [%s]", backupName, namespace))
+						break
+					}
+					mu.Lock()
+					defer mu.Unlock()
+					userBackupMap[user][backupName] = namespace
+				}
+				for _, namespace := range appNamespaces {
+					for i := 0; i < numberOfBackups; i++ {
+						backupName := fmt.Sprintf("%s-%s-%d-%v", BackupNamePrefix, namespace, i, randomSuffix)
+						wg.Add(1)
+						go createBackup(backupName, namespace)
+					}
+				}
+				wg.Wait()
+				log.Infof("The list of user backups taken are: %v", userBackupMap)
+			})
+			Step(fmt.Sprintf("Create schedule policy from the user %s", user), func() {
+				log.InfoD(fmt.Sprintf("Creating schedule policy from the user %s", user))
+				nonAdminCtx, err := backup.GetNonAdminCtx(user, commonPassword)
+				log.FailOnError(err, "failed to fetch user %s ctx", user)
+				userSchedulePolicyName := fmt.Sprintf("%s-%v", "periodic", randomSuffix)
+				userSchedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, userSchedulePolicyInterval, 5)
+				userSchedulePolicyCreateRequest := &api.SchedulePolicyCreateRequest{
+					CreateMetadata: &api.CreateMetadata{
+						Name:  userSchedulePolicyName,
+						OrgId: orgID,
+					},
+					SchedulePolicy: userSchedulePolicyInfo,
+				}
+				_, err = Inst().Backup.CreateSchedulePolicy(nonAdminCtx, userSchedulePolicyCreateRequest)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation schedule policy %s", userSchedulePolicyName))
+				userSchedulePolicyUID, err := Inst().Backup.GetSchedulePolicyUid(orgID, nonAdminCtx, userSchedulePolicyName)
+				log.FailOnError(err, "failed to fetch schedule policy uid %s of user %s", userSchedulePolicyName, user)
+				userSchedulePolicyMap[user] = map[string]string{userSchedulePolicyUID: userSchedulePolicyName}
+			})
+			Step(fmt.Sprintf("Take schedule backup of applications from the user %s", user), func() {
+				log.InfoD(fmt.Sprintf("Taking schedule backup of applications from the user %s", user))
+				nonAdminCtx, err := backup.GetNonAdminCtx(user, commonPassword)
+				log.FailOnError(err, "failed to fetch user %s ctx", user)
+				userScheduleName := fmt.Sprintf("backup-schedule-%v", randomSuffix)
+				for backupLocationUID, backupLocationName := range userBackupLocationMap[user] {
+					for schedulePolicyUID, schedulePolicyName := range userSchedulePolicyMap[user] {
+						_, err = CreateScheduleBackupWithValidation(nonAdminCtx, userScheduleName, SourceClusterName, backupLocationName, backupLocationUID, scheduledAppContexts, make(map[string]string), orgID, "", "", "", "", schedulePolicyName, schedulePolicyUID)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation and validation of schedule backup with schedule name [%s]", schedulePolicyName))
+						break
+					}
+					break
+				}
+				userScheduleNameMap[user] = userScheduleName
+			})
+		}
```

Comment:
I might be missing a step to filter the objects using user, verify the objects from selected user is listed. @ak-px

Reply:
@kphalgun-px , As mentioned please add that step.